cluster/gce/gci/configure-helper.sh

```diff
@@ -733,14 +743,15 @@ EOF
     use_cloud_config="true"
     if [[ -n "${NODE_TAGS:-}" ]]; then
       # split NODE_TAGS into an array by comma.
-      IFS=',' read -r -a node_tags <<< ${NODE_TAGS}
+      IFS=',' read -r -a node_tags <<< "${NODE_TAGS}"
     else
-      local -r node_tags="${NODE_INSTANCE_PREFIX}"
+      local -r node_tags
+      node_tags=${NODE_INSTANCE_PREFIX}
```

Comment:
style nit
```suggestion
      node_tags="${NODE_INSTANCE_PREFIX}"
```

Reply:
Ah, yes, the superfluous assignment quotes.